### PR TITLE
fix a small mistake in tst_timesig.cpp (follow-up for PR #5330)

### PR DESCRIPTION
**Pull request by Howard-C**
`4` -> `VOICES`

EDIT: Now `staff2track(1)`.

### CODE REVIEW

File: mtest/libmscore/timesig/tst_timesig.cpp [hunk 1]
```diff
@@ -224,7 +224,7 @@ void TestTimesig::timesig08()
 
       Measure* m1 = score->firstMeasure();
       Segment* seg = m1->findSegment(SegmentType::TimeSigAnnounce, m1->endTick());
-      Element* el = seg->element(4); // stave no.2
+      Element* el = seg->element(VOICES); // stave no.2
```
Comment (dmitrio95) [inline mtest/libmscore/timesig/tst_timesig.cpp]: Another option: `staff2track(1)` :)
https://github.com/musescore/MuseScore/blob/e44c823c070187fb29a49f21fb2793bda6b072f0/libmscore/mscore.h#L75

Comment (Howard-C) [reply mtest/libmscore/timesig/tst_timesig.cpp]: But isn't this `<< 2` taking advantage of `VOICES` being 4? What if it's 5 or else? Speaking of this, I think directly putting `VOICES` is even more correct than this.

Comment (Howard-C) [reply mtest/libmscore/timesig/tst_timesig.cpp]: Anyway, changed to `staff2track(1)`.

Comment (dmitrio95) [reply mtest/libmscore/timesig/tst_timesig.cpp]: > But isn't this `<< 2` taking advantage of `VOICES` being 4

Yes, but this is (or at least was intended to be) a commonly used function to convert between staff indices and track numbers, and its definition would definitely change if we ever decide to switch voices number or go to another track numbering scheme at all. That is why I mentioned this. Still this conversion is not really likely as too much depends on it so this is definitely not very important issue.

**Pull request merged**